app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comment
+  has_many :like
+  belongs_to :author, class_name: 'User'
+  after_save :update_posts_counter
```

Comment:
**_as mentioned by the previous reviewer:_** 

- [ ] Please fix the name of the other model all over your codebase, wherever you have used a singular name for setting up a has_many relation, currently you only fixed the user modal where you still have the issue in the post model

Reply:
This comment has been resolved, thanks.